### PR TITLE
fix(datasource_django): proxy models are not added to datasource

### PR DESCRIPTION
**Pull request by jbarreau**

### CODE REVIEW

**Comment by jbarreau** [comment]
Task linked: [CU-86bwu2c8j 🔥🔥🔥 P1 - Django onboarding](https://app.clickup.com/t/86bwu2c8j)

File: src/datasource_django/tests/test_project_datasource/test_app/models.py
Comment (nicolasalexandre9) [inline src/datasource_django/tests/test_project_datasource/test_app/models.py:1]: Is it possible to declare a relation into the proxy model ?

Comment (jbarreau) [reply src/datasource_django/tests/test_project_datasource/test_app/models.py:1]: No, after a auick try, `python manage.py check` return :
> ERRORS:
?: (models.E017) Proxy model 'DjangoAuthUser' contains model fields.

**Review by nicolasalexandre9 — APPROVED** [review]
LGTM 👍

**Pull request merged**